methods/eps_fhir/api_request_body_generators.py

```diff
@@ -462,3 +464,124 @@ def generate_group_identifier(prescription_order_number):
             "value": prescription_order_number,
         },
     }
+
+
+def generate_return(nhs_number, short_prescription_id):
```

Comment:
This Task might be better treated in the same way as the Bundle, so a more top-level resource.

That way you can create the Practitioner, PractitionerRole and Organization using your existing functions in this file.